plugins/GrapesJsBuilderBundle/Model/GrapesJsBuilderModel.php

```diff
@@ -68,7 +68,8 @@ public function addOrEditEntity(Email $email): void
             $grapesJsBuilder->setEmail($email);
         }
 
-        if ($this->requestStack->getCurrentRequest()->request->has('grapesjsbuilder')) {
+        $currentRequest = $this->requestStack->getCurrentRequest();
+        if ($currentRequest && $this->requestStack->getCurrentRequest()->request->has('grapesjsbuilder')) {
```

Comment:
```suggestion
        if ($currentRequest && $currentRequest->request->has('grapesjsbuilder')) {
```
No need to call the `getCurrentRequest()` function twice.

Reply:
fixed